### PR TITLE
Improve packages readme

### PR DESCRIPTION
**Pull request by jamesblasco**
fix: minor issues

### CODE REVIEW

File: packages/stripe/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.1+1
+
+ - Minor fix
```
Comment (remonh87) [inline packages/stripe/CHANGELOG.md:3]: maybe add the issue number I would like a bit more elaborate explanation

File: packages/stripe/README.md [hunk 2]
```diff
@@ -104,4 +121,4 @@ The example app offers examples on how to use these methods.
 
 ## Known issues
 
-- the Card field on Android has focus issues - it's being tracked in [#14](https://github.com/flutter-stripe/flutter_stripe/issues/14) and related to issues in the Flutter framework
\ No newline at end of file
+- the Card field on Android has focus issues - it's being tracked in [#14](https://github.com/flutter-stripe/flutter_stripe/issues/14) and related to issues in the Flutter framework
```
Comment (remonh87) [inline packages/stripe/README.md:124]: Maybe we should kindly advise to use the paymentsheet?

File: packages/stripe/pubspec.yaml [hunk 3]
```diff
@@ -11,10 +11,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  stripe_android: ^0.0.1
-  stripe_ios: ^0.0.1
-  stripe_platform_interface: ^0.0.1
-
+  stripe_android: ^0.0.1+1
```
Comment (remonh87) [inline packages/stripe/pubspec.yaml:14]: shouldn't these become 1.0.1+1 as well just to try to keep things uniform?

**Review by remonh87 — APPROVED** [review]
Looks good just some small comments

File: packages/stripe/pubspec.yaml (hunk 3 above)
Comment (jamesblasco) [reply packages/stripe/pubspec.yaml:14]: Yeah, I agree, let's do it for the next release. I already published this one

**Review by jonasbark — APPROVED** [review]

**Pull request merged**